.github/workflows/nix-ci-aarch64.yml

```diff
@@ -2,13 +2,20 @@ name: Nix aarch64 builds
 
 on:
   workflow_dispatch: # allows manual triggering
+  schedule:
+    # Rebuild daily rather than on every push because QEMU is expensive (e.g.
+    # 1.5h instead of minutes with the cold cache).
+    #
+    # randint(0, 59), randint(0, 23)
+    - cron: '26 12 * * *'
+  # But also rebuild if we touched any of the Nix expressions:
```

Comment:
That's a great compromise.

Reply:
@ggerganov would this be sufficient to re-enable the aarch64 workflow?

Reply:
Ok, I will reenable now